packages/terra-form/src/Control.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import Input from './Input';
+
+import './Control.scss';
+
+const propTypes = {
+  /**
+   * The input type
+   */
+  type: PropTypes.oneOf(['checkbox', 'radio']).isRequired,
+  /**
+   * The checked property of the Input element. Use this to generate a controlled Control Element
+   */
+  checked: PropTypes.bool,
+  /**
+   * The checked property of the Input element. Use this to generate an uncontrolled Control Element
+   */
+  defaultChecked: PropTypes.bool,
+  /**
+   * id of the input field
+   */
+  id: PropTypes.string,
+  /**
+   * Additional attributes for the input object
+   */
+  inputAttrs: PropTypes.object,
+  /**
+   * Whether the form element is inline or not
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Text of the label
+   */
+  labelText: PropTypes.node,
+  /**
+   * Additional objects for the text object
+   */
+  labelTextAttrs: PropTypes.object,
```

Comment:
That's actually covered by customProps, as the main container is a label. If you think that's confusing though, I can change customProps to be labelAttrs.